actionpack/lib/action_controller/log_subscriber.rb

```diff
@@ -26,6 +26,8 @@ def process_action(event)
         end
         message = "Completed #{status} #{Rack::Utils::HTTP_STATUS_CODES[status]} in #{event.duration.round}ms"
         message << " (#{additions.join(" | ".freeze)})" unless additions.blank?
+        message << "\n\n" if Rails.env.development?
```

Comment:
This is coupling Action Pack with Railties we should check if `Rails.env` is defined before calling it.